### ISSUE TITLE
Syntax error in Astro project as React Islands (easy fix)

### ISSUE THREAD

**Issue by isimmons** [pqoqubbw/icons#15]
I tried the icons in an Astro project.

1. `npx astro add react`
2. `npm i framer-motion`
3.  download desired icons to src/icons or where ever you want to keep them

The error when Astro compiles: `[vite] The requested module 'framer-motion' does not provide an export named 'Variants'`
Or the same error for  'Transition' for components that import that type.

To fix it I explicitly used the type keyword. 

```ts
import {
  motion,
  type Transition,
  useAnimation,
  type Variants,
} from 'framer-motion'
```
I could be wrong but I don't think including the type keyword will cause issues in cases where it's not required so it shouldn't hurt to put it in there on all the components which will make these icons compatible with Astro/React.

**Comment by pqoqubbw** [comment]
fix added 6ce1b70c1711c042afe5d6616ca7c477fcd904dd, could you check it please?

**Comment by isimmons** [comment]
@pqoqubbw I went through them all visually double checking and added several to my Astro project. It all looks good to me.

I noticed a strange thing on Github though. When I open calendar-cog.tsx, download.tsx, or fingerprint.tsx they have partially lost syntax highlighting. I don't see any reason why. Maybe just some bug on github? 

I made sure to download and test those too and they are working great.

I'll close but let me know if I can test anything else for you and I'll try.

Thanks
